test/rxd/ecs/test_ecs_include_flux.py

```diff
@@ -0,0 +1,46 @@
+import pytest
```

Comment:
If this is sufficiently similar to test_ecs_example maybe both files can be merged into one?

Reply:
I think it is quite different, test_ecs_example is an example model whereas test_ecs_include_flux is just placing node_flux into the ecs in 3 different ways.